### DISCUSSION TITLE
Airflow 2.0 and disable RBAC

### DISCUSSION THREAD

**Discussion by usernametakenforever** [apache/airflow#13749] [Q&A] [answered]
I have noticed that in airflow 2.0.0, RBAC is enabled by default and new `webserver_config.py `is generated automatically. Is it possible for me to use my webserver and ldap settings in `airflow.cfg` from older version (1.10.12)? CPU usage on the host just went up considerably and I believe it may be due to RBAC in webserver. Please let me know if you need more informaiton.

**Comment by kaxil** [comment, marked as answer]
Airflow 2.0 uses RBAC UI and it is the only UI. The old flask-admin based UI has been removed in 2.0.0

**Reply by dstandish** [reply]
is it possible to diable auth within rbac ui? or we simply must create a user and use it to log in?

**Reply by kaxil** [reply]
You can uncomment this line in Airflow 2.0.1 and change `Public` to Admin:

https://github.com/apache/airflow/blob/9f14724d160f12a1315cfcd7e557eae820de8498/airflow/config_templates/default_webserver_config.py#L52-L53

**Reply by dstandish** [reply]
nice! thanks.

**Reply by dstandish** [reply]
ok i actually got it working with this setting

```
AUTH_ROLE_PUBLIC = 'Admin'
```

but there is still the login button at top right...  which will probably be a bit confusing for users

**Reply by kaxil** [reply]
Agree, worth creating an issue with FAB (https://github.com/dpgaspar/Flask-AppBuilder) I think

**Comment by Bashtel** [comment]
We also noticed an increase in CPU usage after the upgrade, but found that it was caused by the scheduler running the DagFileProcessor non-stop. I tweaked the corresponding config keys (namely "processor_poll_interval" and "min_file_process_interval"). Not sure if it was a fuck up in the upgrade process by me or the upgrade introduced new configs there.

**Reply by kaxil** [reply]
`min_file_process_interval` has been changed to `30` from `0` since 2.0.1

https://github.com/apache/airflow/blob/master/UPDATING.md#default-scheduler-min_file_process_interval-is-changed-to-30
